### PR TITLE
add randseed.

### PR DESCRIPTION
**Pull request by aaronUniontech**
Signed-off-by: zhangya <zhangya@uniontech.com>

### CODE REVIEW

**Review by AndreasFuchsSIT — APPROVED** [review]

File: src/tpm2-tss-engine-rand.c [hunk 1]
```diff
@@ -40,6 +40,46 @@
...
+    TSS2_RC r;
+
+    r = esys_ctx_init(&esys_ctx);
+    ERRchktss(rand_seed, r, goto end);
+
+    TPM2B_SENSITIVE_DATA stir;
+    size_t offset = 0;
+    char *cur_data = (char*)seed;
+
+    static const size_t tpm_random_stir_max_size = 128;
```
Comment (AndreasFuchsSIT) [inline src/tpm2-tss-engine-rand.c:60]: Unfortunately there is a trailing whitespace here.

**Review by AndreasFuchsSIT — CHANGES_REQUESTED** [review]
One last thing from our CI, then I'll merge and roll:
```
src/tpm2-tss-engine-rand.c:60: trailing whitespace.
+    static const size_t tpm_random_stir_max_size = 128; 
```

File: src/tpm2-tss-engine-rand.c (hunk 1 above)
Comment (aaronUniontech) [reply src/tpm2-tss-engine-rand.c:60]: i fix it now.

Comment (AndreasFuchsSIT) [reply src/tpm2-tss-engine-rand.c]: Sorry, but I need this to be one commit.
So you can just squash those two commits together and force-push them.

**Pull request closed without merging**